### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by tygyh**
Fixes #

Changes proposed in this request:
- Remove some redundant bool comvertions

### CODE REVIEW

File: src/engine/client.cpp [hunk 1]
```diff
@@ -199,8 +199,8 @@ ICOMMAND(0, disconnect, "i", (int *force), trydisconnect(*force!=0));
 ICOMMAND(0, lanconnect, "is", (int *a, char *pwd), connectserv(NULL, *a, pwd));
 ICOMMAND(0, localconnect, "i", (int *n), localconnect(*n ? false : true));
 
-ICOMMAND(0, isonline, "", (), intret(curpeer ? 1 : 0));
-ICOMMAND(0, isconnected, "ii", (int *a, int *b), intret(connected(*a==0, *b==0) ? 1 : 0));
+ICOMMAND(0, isonline, "", (), intret(curpeer));
```
Comment (q009) [inline src/engine/client.cpp:202]: The idea here was to limit the return value to either 1 or 0.

File: bin/redeclipse.app/Contents/Frameworks/SDL2.framework/Versions/A/Headers/SDL_endian.h [hunk 2]
```diff
@@ -158,7 +158,7 @@ SDL_Swap64(Uint64 x)
         Uint64 u;
     } v;
     v.u = x;
-  __asm__("bswapl %0 ; bswapl %1 ; xchgl %0,%1": "=r"(v.s.a), "=r"(v.s.b):"0"(v.s.a),
+  __asm__("bswapl %0; bswapl %1; xchgl %0,%1": "=r"(v.s.a), "=r"(v.s.b):"0"(v.s.a),
```
Comment (q009) [inline bin/redeclipse.app/Contents/Frameworks/SDL2.framework/Versions/A/Headers/SDL_endian.h:161]: It appears you just blindly replaced all " ;" with just ";".  It's asking for trouble and might mess with syntax of various scripts and whatnot

File: src/engine/command.cpp [hunk 3]
```diff
@@ -4865,7 +4865,7 @@ ICOMMAND(0, error, "C", (char *s), conoutf("\fr%s", s));
             for(int i = 2; i < numargs && val; i++) val = func(args[i-1].s, args[i].s) op 0; \
         } \
         else val = (numargs > 0 ? args[0].s[0] : 0) op 0; \
-        intret(val ? 1 : 0); \
+        intret(val); \
```
Comment (q009) [inline src/engine/command.cpp:4868]: Once again, altering the return value of functions. As well as in many other places.

File: src/engine/renderparticles.cpp [hunk 4]
```diff
@@ -1058,7 +1058,7 @@ struct loopprimitiverenderer : listrenderer<loopprimitive>
         glBindTexture(GL_TEXTURE_2D, blanktexture->id);
         gle::colorub(p->color.r, p->color.g, p->color.b, uchar(p->blend*blend));
         gle::begin(p->fill ? GL_TRIANGLE_FAN : GL_LINE_LOOP);
-        loopi(15 + (p->fill ? 1 : 0))
+        loopi(15 + (p->fill))
```
Comment (q009) [inline src/engine/renderparticles.cpp:1061]: This is very, very wrong. Don't use blind string replace.

**Review by q009 — CHANGES_REQUESTED** [review]
Don't use blind string replace.

**Comment by qreeves** [comment]
Dangling PR without a response to changes requested. Closing.

**Pull request closed without merging**